spaghetti/tests/test_network.py

```diff
@@ -493,7 +493,7 @@ def test_element_as_gdf(self):
         _, tree = ntw.allneighbordistances(points1, points2, gen_tree=True)
         paths = ntw.shortest_paths(tree, points1, pp_dest=points2)
         paths_gdf = spaghetti.element_as_gdf(ntw, routes=paths)
-        observed_origins = paths_gdf["O"].nunique()
+        observed_origins = paths_gdf["id"].map(lambda x: x[0]).nunique()
```

Comment:
Avoid `map`-based loop by using `numpy.unique` over the slice of an array.

Reply:
So here I changed the output columns to avoid redundancy:

where previously there were `["O", "D", "id"]` columns:
```paths_gdf
    O    D      id                                           geometry
0   0    0  (0, 0)  LINESTRING (0.50000 1.00000, 1.00000 1.00000, ...
1   0    1  (0, 1)  LINESTRING (0.50000 1.00000, 1.00000 1.00000, ...
2   1    0  (1, 0)  LINESTRING (2.00000 2.50000, 2.00000 2.00000, ...
3   1    1  (1, 1)  LINESTRING (2.00000 2.50000, 2.00000 2.00000, ...
```

now there is only "id": 
```paths_gdf
       id                                           geometry
0  (0, 0)  LINESTRING (0.50000 1.00000, 1.00000 1.00000, ...
1  (0, 1)  LINESTRING (0.50000 1.00000, 1.00000 1.00000, ...
2  (1, 0)  LINESTRING (2.00000 2.50000, 2.00000 2.00000, ...
3  (1, 1)  LINESTRING (2.00000 2.50000, 2.00000 2.00000, ...
```

Reply:
I am now wondering if this should be a depreciation like the #722 stuff...

Reply:
`map` with lambda will be always slower than numpy ops.

Reply:
ignore this, I didn't realise we're in a test suite only.